### ISSUE TITLE
0.18: app dies on rotate

### ISSUE THREAD

**Issue by goldencut** [AdrienPoupa/VinylMusicPlayer#80]
Upgraded .17 to latest and when I rotated the phone Android told me that app had quit. Tried multiple times - same. Went back to .17 and it works fine. Andr 4.4.2. 

PS. VMP is awesome.

**Comment by gkeegan** [comment]
Crashes on putting it into split screen too

**Comment by AdrienPoupa** [comment]
Hi,

Do you have some logs to share? It's working fine for me on LineageOS 15.1. It may have to do with android 4.4, which is ancient at this point

**Comment by gkeegan** [comment]
I'm on Android Oreo, how exactly would I get logs of the crash?

**Comment by AdrienPoupa** [comment]
I think this could be helpful: https://www.makeuseof.com/tag/get-logcat-reporting-bugs-android/

Thanks!

**Comment by LightWayUp** [comment]
I can confirm this issue too. Here's the link to the entire logcat file, in case you need it : https://drive.google.com/open?id=12eBbCw3BVuAuSz-DyNSGuYwO3q3JXTK2

But I believe this is the more critical part of the logs :
```
[ 09-26 02:17:44.329  9189: 9189 D/AndroidRuntime ]
Shutting down VM


--------- beginning of crash
[ 09-26 02:17:44.332  9189: 9189 E/AndroidRuntime ]
FATAL EXCEPTION: main
Process: com.poupa.vinylmusicplayer, PID: 9189
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.poupa.vinylmusicplayer/com.poupa.vinylmusicplayer.ui.activities.AlbumDetailActivity}: android.view.InflateException: Binary XML file line #14: Binary XML file line #14: Error inflating class com.kabouzeid.gramophone.views.StatusBarView
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2684)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2751)
        at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:4556)
        at android.app.ActivityThread.-wrap19(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1502)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6186)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
Caused by: android.view.InflateException: Binary XML file line #14: Binary XML file line #14: Error inflating class com.kabouzeid.gramophone.views.StatusBarView
Caused by: android.view.InflateException: Binary XML file line #14: Error inflating class com.kabouzeid.gramophone.views.StatusBarView
Caused by: java.lang.ClassNotFoundException: Didn't find class "com.kabouzeid.gramophone.views.StatusBarView" on path: DexPathList[[zip file "/data/app/com.poupa.vinylmusicplayer-1/base.apk"],nativeLibraryDirectories=[/data/app/com.poupa.vinylmusicplayer-1/lib/arm, /system/lib, /vendor/lib]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:380)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at android.view.LayoutInflater.createView(LayoutInflater.java:609)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:787)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:727)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:858)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:821)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:861)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:821)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:518)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:426)
        at com.poupa.vinylmusicplayer.ui.fragments.player.flat.FlatPlayerFragment.onCreateView(Unknown Source)
        at android.support.v4.app.Fragment.performCreateView(Unknown Source)
        at android.support.v4.app.FragmentManagerImpl.moveToState(Unknown Source)
        at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(Unknown Source)
        at android.support.v4.app.FragmentManagerImpl.moveToState(Unknown Source)
        at android.support.v4.app.FragmentManagerImpl.dispatchStateChange(Unknown Source)
        at android.support.v4.app.FragmentManagerImpl.dispatchActivityCreated(Unknown Source)
        at android.support.v4.app.FragmentController.dispatchActivityCreated(Unknown Source)
        at android.support.v4.app.FragmentActivity.onStart(Unknown Source)
        at android.support.v7.app.AppCompatActivity.onStart(Unknown Source)
        at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1249)
        at android.app.Activity.performStart(Activity.java:6701)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2647)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2751)
        at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:4556)
        at android.app.ActivityThread.-wrap19(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1502)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6186)

[ 09-26 02:17:44.332  9189: 9189 E/AndroidRuntime ]
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)


[ 09-26 02:17:44.345  1608: 3274 W/ActivityManager ]
  Force finishing activity com.poupa.vinylmusicplayer/.ui.activities.AlbumDetailActivity
```

**Comment by AdrienPoupa** [comment]
Thanks for the logs!

My mistake, it should be fixed now.

**Comment by LightWayUp** [comment]
@AdrienPoupa Sorry for bothering you, I know the issue is fixed in commit b552dc0 . However, there hasn't been any new releases yet after the commit, so the app still crashes every time the device is in landscape mode or split screen is activated. I would be grateful if you can create a new release soon, and publish it to Google Play Store and F-Droid 😛.
